samples/ClassroomLabs/Modules/Library/Tests/LabAccount.test.ps1

```diff
@@ -0,0 +1,30 @@
+[cmdletbinding()]
+Param()
+Import-Module $PSScriptRoot\..\Az.LabServices.psm1
+
+$rgName     = 'TestPerm'
+$rgLocation = 'West Europe'
+$laName     = 'LA' + (Get-Random)
+
+Describe 'Lab Account Crud' {
+        It 'Can create a Lab Account' {
+
+            if(-not (Get-AzResourceGroup -ResourceGroupName $rgName -EA SilentlyContinue)) {
+                New-AzResourceGroup -ResourceGroupName $rgName -Location $rgLocation | Out-null
+                Write-Verbose "$rgname resource group didn't exist. Created it."
+            }
+            
+            $la  = New-AzLabAccount -ResourceGroupName $rgName -LabAccountName $laName
+            Write-Verbose "$laName lab account created or found."
+            
+            $la | Should -Not -BeNullOrEmpty
+        }
+
+        It 'Can remove Lab Account' {
+
+            $la = Get-AzLabAccount -ResourceGroupName $rgName -LabAccountName $laName
+            $la | Remove-AzLabAccount
+
+            Get-AzLabAccount -ResourceGroupName $rgName -LabAccountName $laName | Should -Not -BeNullOrEmpty 
```

Comment:
The lab account was just removed - shouldn't we validate "Should -BeNullOrEmpty" instead of "Should -Not -BeNullOrEmpty"?

Reply:
My intuition is that it was working before because it would get the lab account before the removal was complete. I can either make the removal wait for completion or remove the check. I went for the latter as making the removal synchronous is not trivial and perhaps not correct either (aka in practical code, why wait for it?). In any case, if you consider it wrong, reopen the conversation.